public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -33,6 +67,10 @@ export const ReceiversTable: FC<Props> = ({ config, alertManagerName }) => {
   const isVanillaAM = isVanillaPrometheusAlertManagerDataSource(alertManagerName);
   const permissions = getNotificationsPermissions(alertManagerName);
   const grafanaNotifiers = useUnifiedAlertingSelector((state) => state.grafanaNotifiers);
+  const contactPointsStateRequest = useUnifiedAlertingSelector((state) => state.contactPointsState);
+  const { result: contactPointsState } = (alertManagerName && contactPointsStateRequest) || initialAsyncRequestState;
+  const receivers = contactPointsState?.receivers ?? {};
+  const errorStateAvailable = Object.keys(receivers).length > 0; // this logic can change depending on how we implement this in the BE
```

Comment:
This code looks quite similar to what we do inside `ReceiverHealth`.
Maybe we could e.g. create a custom hook like `useContactPointErrors` here that provides the errors and then only pass the receiver-specific error as a prop to the `ReceiverHealth` component. This way we could decouple `ReceiverHealth` component from redux

Reply:
💯